src/database/migration/file/index.ts

```diff
@@ -5,14 +5,16 @@ import MigrationVersion2 from './migration_version_2';
 import MigrationVersion3 from './migration_version_3';
 import MigrationVersion4 from './migration_version_4';
 import MigrationVersion5 from './migration_version_5';
+import MigrationVersion6 from './migration_version_6';
 
 const allMigrations: Migration[] = [
   MigrationVersion0,
   MigrationVersion1,
   MigrationVersion2,
   MigrationVersion3,
   MigrationVersion4,
-  MigrationVersion5
+  MigrationVersion5,
+  MigrationVersion6
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `MigrationVersion6` to the `allMigrations` array seems fine. However, ensure that the `MigrationVersion6` module is correctly implemented and exported before importing it here. Also, make sure that this new migration version is handled properly in your database migration logic.
```diff
+ import MigrationVersion6 from './migration_version_6';
...
+   MigrationVersion6
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->